### PR TITLE
DOCSP-41816 updates tools version support to include 8.0

### PR DESCRIPTION
**Pull request by jwilson-mdb**
## DESCRIPTION
Updates version support to include v8.0

## STAGING
The updated file is used to show 8.0 compatibility for:

- [General Installation](https://deploy-preview-174--docs-commandline-tools.netlify.app/installation/installation/#mongodb-server-compatibility) and the Windows/Mac/Linux sub-pages
- [`mongodump` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongodump/mongodump-compatibility-and-installation/#mongodb-server-compatibility)
- [`mongorestore` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongorestore/mongorestore-compatibility-and-installation/#mongodb-server-compatibility)
- [`bsondump` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/bsondump/bsondump-compatibility-and-installation/#mongodb-server-compatibility)
- [`mongoimport` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongoimport/mongoimport-compatibility-and-installation/#mongodb-server-compatibility)
- [`mongoexport` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongoexport/mongoexport-compatibility-and-installation/#mongodb-server-compatibility)
- [`mongostat` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongostat/mongostat-compatibility-and-installation/#mongodb-server-compatibility)
- [`mongotop` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongotop/mongotop-compatibility-and-installation/#mongodb-server-compatibility)
- [`mongofiles` compatibility](https://deploy-preview-174--docs-commandline-tools.netlify.app/mongofiles/#mongodb-server-compatibility)

## JIRA
https://jira.mongodb.org/browse/DOCSP-41816

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/extracts-compatibility.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ content: |
    {+dbtools+} version ``{+release+}`` supports the following versions
    of the MongoDB server:
 
+   - MongoDB 8.0
    - MongoDB 7.0
    - MongoDB 6.0
    - MongoDB 5.0
```
Comment (ajhuh-mdb) [inline source/includes/extracts-compatibility.yaml:10]: [non-blocking question] Should we remove v4.2 and v4.4 since they're EOL'd?

**Review by ajhuh-mdb — APPROVED** [review]
LGTM with one non-blocking question!

File: source/includes/extracts-compatibility.yaml (hunk 1 above)
Comment (jwilson-mdb) [reply source/includes/extracts-compatibility.yaml:10]: Thanks @ajhuh-mdb - good catch. I'll bring it up with the tech reviewer and make the change if they give the okay.

Comment (tdq45gj) [reply source/includes/extracts-compatibility.yaml:10]: We don't need to remove them because the DB tools still support v4.2 and v4.4 at the moment.

**Review by tdq45gj — APPROVED** [review]
LGTM, thanks!

**Pull request merged**